bloom/src/bloom.rs

```diff
@@ -308,9 +308,16 @@ mod test {
         );
     }
 
+    fn generate_random_hash() -> Hash {
```

Comment:
lol... `Hash::new_unique()` is broken.  it should be _hashing_ the counter, not using it straight up :upside_down_face:

Reply:
I can try changing it to that and see if the bloom filter test passes. On the flipside, and I might be understanding this wrong, but it seems like the test is meant for random hashes.

Reply:
nah don't mess with it here